File: CS/WindowsFormsApplication662/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraTab.Registrator;


namespace WindowsFormsApplication662 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            PaintStyleCollection.DefaultPaintStyles.Add(new MySkinViewInfoRegistrator());
            xtraTabControl1.PaintStyleName = "MyStyle";
        }
    }
}

File: VB/WindowsFormsApplication662/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraTab.Registrator


Namespace WindowsFormsApplication662
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
            PaintStyleCollection.DefaultPaintStyles.Add(New MySkinViewInfoRegistrator())
            xtraTabControl1.PaintStyleName = "MyStyle"
        End Sub
    End Class
End Namespace
